packages/create-docusaurus/bin/index.js

```diff
@@ -1,17 +1,19 @@
-#!/usr/bin/env node
+#!/usr/bin/env node --experimental-specifier-resolution=node
```

Comment:
Some day, this flag is may be removed or renamed and completely break everything.  I feel like it's not a good idea to hard-code it.

Reply:
Yes, this is just to make sure things work. Soon after things start working we will try to explicitly specify file names & extensions (especially when TS 4.6 has better support for this)